test/CSAccounting.t.sol

```diff
@@ -3907,9 +4037,13 @@ contract CSAccountingPullFeeRewardsTest is CSAccountingBaseTest {
 
         assertEq(bondSharesAfter, bondSharesBefore + feeShares);
         assertEq(totalBondSharesAfter, totalBondSharesBefore + feeShares);
+
+        // mock call to avoid invariant failure
+        // this not distributed because of feeDistributor mock
+        stETH.mintShares(address(accounting), feeShares);
```

Comment:
So maybe it was a mistake to use a mock call to the `feeDistributor`, because we expect a side effect for the tests. I suggest making it a simple contract implementing the side effect.

Reply:
fixed